modules/ROOT/pages/security/ssl-framework.adoc

```diff
@@ -271,7 +271,7 @@ Each policy needs to be explicitly enabled by setting:
 === Configure SSL over Bolt
 
 Bolt protocol is based on the link:https://neo4j.com/docs/bolt/current/packstream/[PackStream serialization] and supports the Cypher type system, protocol versioning, authentication, and TLS via certificates.
-For Neo4j clusters, Bolt provides smart client routing with load balancing and failover.
+For Neo4j clusters, Bolt provides smart client routing with load balancing and failover through port 7688.
```

Comment:
This addition feels slightly misleading, though technically accurate in some senses. 'Smart client routing' and 'load balancing' do not use server side routing, and 'failover' doesn't either in most cases (if a secondary or follower fails a query, send it to a different one mentioned in the routing table). Server-side routing is an additional Bolt port that is only for use within the cluster.

Maybe something along the lines of 'When server side routing is enabled, there will be an additional Bolt port open on 7688 for use only within the cluster, using all the same settings as the external Bolt port'?

Reply:
Thanks Nick. I have added your suggestion